pkg/cluster/kubernetes/resource/load.go

```diff
@@ -170,3 +178,7 @@ func ParseMultidoc(multidoc []byte, source string) (map[string]KubeManifest, err
 	}
 	return objs, nil
 }
+
+func shouldDecrypt(path string) bool {
+	return strings.HasSuffix(path, ".enc.yaml") || strings.HasSuffix(path, ".enc.yml")
```

Comment:
Shouldn't this be using the suffix provided to the `--sops-suffix` flag?

Reply:
Yes. It has been corrected in 8d45ecda8c8b617cd6276f4f6c977541a66b06f7